libclc/CMakeLists.txt

```diff
@@ -137,10 +137,6 @@ if( llvm-spirv_exe )
   list( APPEND LIBCLC_TARGETS_ALL  spirv-mesa3d- spirv64-mesa3d- )
 endif()
 
-if( LIBCLC_TARGETS_TO_BUILD STREQUAL "all" )
-  set( LIBCLC_TARGETS_TO_BUILD ${LIBCLC_TARGETS_ALL} )
-endif()
-
 list( SORT LIBCLC_TARGETS_TO_BUILD )
```

Comment:
Do we need to move this `SORT` too? I doubt it's important for correctness, but if the targets under `all` aren't sorted, then this changes their order.

Reply:
Yes, I've moved it as well.